even_fib_nums.js

```diff
@@ -7,10 +8,22 @@
  */
 function _sumFibs( maxFibValue ) {
   var sum = 0;
-
+  let newSum = 0;
+  let a = 0;
+  let b = 1;
+  for(let i = 0; i < maxFibValue; i = sum){
+    sum = a + b;
+    a = b;
+    b = sum;
+      if(sum % 2 === 0){
+        newSum += sum;
```

Comment:
See if you can refactor to maintain a single sum value, and eliminate the need for a second sum variable.